package NProtoConfigUt;

message TTestConfig {
    repeated string DataFile = 1;

    enum EMode {
        READ = 1;
        WRITE = 2;
    }

    optional EMode Mode = 2;

    message TNetwork {
        optional string Host = 1;
        optional uint32 Port = 2;
    }

    optional TNetwork Network = 3;
}